accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -325,6 +354,30 @@ not a subtype of the class type associated with the class in which it is defined
 (specifically, it is an error to return a nullable type from a factory
 constructor for any class other than `Null`).
 
+It is an error if any case of a switch statement except the last case (the
+default case if present) **may complete normally**.  The previous syntactic
+restriction requiring the last statement of each case to be one of a delimited
```

Comment:
Is "delimited" correct here? (Non-native English speaker woes - I associate delimited to *having delimiters*, not just being *limited*, and I'm not sure what those delimiters would be here. Maybe I'm just too limited in my view.)

Reply:
It is a correct usage - derived I think from the sense of the set not being open ended, but instead having fixed boundaries.  But `enumerated` is probably clearer here, so I'll use that.